.ado/scripts/npmAddUser.mjs

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+// @ts-check
+
+import { exec } from "node:child_process";
```

Comment:
I assume you just prefer ES6 modules? I do too, but I wasn't sure if there was a stronger reason.

Reply:
For internal things, there are no reasons to keep writing CJS any longer. If anything, it saves us from converting later.